time/src/main/java/io/spine/time/string/LocalDateTimeStringifier.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2018, TeamDev. All rights reserved.
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.time.string;
+
+import io.spine.time.LocalDateTime;
+import io.spine.time.LocalDateTimes;
+
+/**
+ * The default stringifier for {@link io.spine.time.LocalDateTime LocalDateTime} values.
+ *
+ * @author Alexander Yevsyukov
+ */
+final class LocalDateTimeStringifier extends JtStringifier<LocalDateTime, java.time.LocalDateTime> {
+
+    private static final long serialVersionUID = 0L;
+    private static final LocalDateTimeStringifier INSTANCE = new LocalDateTimeStringifier();
+
+    private LocalDateTimeStringifier() {
+        super("TimeStringifiers.forLocalDateTime()",
```

Comment:
Same as above. That's odd.

Reply:
As discussed vocally, this is the way to show from where a stringifier is called using public API.